packages/uni_app/lib/controller/fetchers/library_occupation_fetcher.dart

```diff
@@ -39,6 +39,7 @@ class LibraryOccupationFetcher {
       }),
     );
 
+    libraryOccupation.sortFloors();
```

Comment:
This is not relevant to this PR, right?
If so, please exclude these changes from the PR.